R/asserts.R

```diff
@@ -257,6 +257,24 @@ assertNotInDataTable <- function(vars, dt, call = sys.call(-1)) {
     }
 }
 
+#' Check if single string not in vector of strings
+#'
+#' @description Checks if all passed vars have not been defined in dt.
+#' @param ... vars Name of variables to check.
+#' @param dt data.table to check for vars. Can also be character vector with
+#' defined variables.
+#' @noRd
+assertNotInVector <- function(var, vec, call = sys.call(-1)) {
+
+  areInVector <- var %in% vec
+  namevar <- deparse(substitute(var))
```

Comment:
`nameVar` and `nameVec` (also for the error function) to stick to the style guide :)

Reply:
Fixed.